spring-boot-actuator/pom.xml

```diff
@@ -139,6 +139,11 @@
 		</dependency>
 		<dependency>
 			<groupId>org.infinispan</groupId>
+			<artifactId>infinispan-core</artifactId>
+			<optional>true</optional>
+		</dependency>
```

Comment:
Indeed. Starting with Infinispan 9 `infinispan-jcache` module has `infinispan-core` as `provided` dependency. So if an end-user application depends on `infinispan-jcache`, it must also depend on `infinispan-core`. More info can be found [here](http://issues.jboss.org/browse/ISPN-6295).